book_programs/chapter_2/buffons_needle.py

```diff
@@ -0,0 +1,134 @@
+# Usage: python3 name_of_program.py <n> 
```

Comment:
The mathematica version has a print boolean parameter which determines whether the results are graphed. The estimate for pi is always printed in the console. So you could run the experiment and get the estimate for pi without seeing the graph.

Reply:
I was thinking about that. I'll look into it. I'm AFK for a while.